python/catkin/builder.py

```diff
@@ -287,17 +288,8 @@ def get_multiarch():
 
 def get_python_install_dir():
     # this function returns the same value as the CMake variable PYTHON_INSTALL_DIR from catkin/cmake/python.cmake
```

Comment:
That's true, but it was related to potentially avoiding every package calling python itself by having the tool compute it once and pass it on the command line.